spec/ndx-anatomical-localization.extensions.yaml

```diff
@@ -67,4 +71,83 @@ groups:
         neurodata_type_inc: VectorData
         doc: "The brain region associated with the localization"
         dtype: text
-        quantity: "?"
\ No newline at end of file
+        quantity: "?"
+
+  - neurodata_type_def: AnatomicalCoordinatesImage
+    neurodata_type_inc: NWBContainer
+    doc: "A class for storing anatomical coordinates as an image"
+    links:
+      - target_type: Space
+        quantity: 1
+        doc: "The space in which the coordinates are defined"
+        name: space
+      - target_type: Image
+        quantity: "?"
+        doc: "The image associated with the coordinates"
+        name: image
+      - target_type: ImagingPlane
+        quantity: "?"
+        doc: "The imaging plane associated with the coordinates"
+        name: imaging_plane
+    attributes:
+      - name: method
+        doc: "The method used to determine the coordinates"
+        dtype: text
+        required: true
+      - name: description
+        doc: "A description of the anatomical coordinates image"
+        dtype: text
+        required: false
+    datasets:
+      - name: x
+        dtype: float32
+        dims:
+          - - width
+            - height
+        shape:
+          - - null
+            - null
+        doc: 2D array containing X coordinates for each pixel (width x height)
+        quantity: 1
+      - name: y
+        dtype: float32
+        dims:
+          - - width
```

Comment:
should we go the other way around here (height, width) thinking on the future?

Although I think that what they have is the orientation of the image.